config.py

```diff
@@ -15,8 +15,13 @@
         # Bot settings
         Validator("log_level", is_type_of=str),
         Validator("bot_token", must_exist=True, is_type_of=str),
+        # Redis settings
+        Validator("redis_host", must_exist=True, is_type_of=str),
+        Validator("redis_port", must_exist=True, is_type_of=int),
```

Comment:
Good start! As part of the MVP, we should also have support for a Redis username and password (and lack thereof). 

(For context: The production network actually does have an authenticated Redis instance for tracking leaderboards)